packages/paste-core/components/progress-steps/stories/horizontal.stories.tsx

```diff
@@ -102,3 +104,22 @@ export const Internationalized: React.FC = () => {
     </ProgressSteps>
   );
 };
+
+export const WithContent = (): React.ReactNode => {
```

Comment:
question: what was the reasoning for adding this story? don't think we need it

Reply:
but let's keep at least 1 horizontal story without `<ProgressStepSeparator>` so we can vrt both ways of doing it on both variants

Reply:
I don't think we need this example, it will need more work and we haven't seen enough example to have it.